docs/deployment.md

```diff
@@ -0,0 +1,42 @@
+## Development
```

Comment:
All of this text is development related, rather than deployment related, and should probably be in the development.md file instead. Deployment would be how we use CI, etc.

Reply:
alright.

Reply:
correction in commit description: The development related text is inside development.md and not in documentation.md
sorry for the mistake.